src/components/VAlert/__snapshots__/VAlert.spec.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`VAlert.vue should be closed by default 1`] = `
 
-<div class="alert error--light"
+<div class="alert undefined"
```

Comment:
Failing snapshots should not be updated unless it's caused by an intentional change.

Reply:
Was a mistake.